### PR TITLE
Le testcase

### LINKED ISSUE #1602: Lees Edwards BC is missing test

### PR DESCRIPTION
**Pull request by bindgens1**
Fixes #1602.

### CODE REVIEW

File: testsuite/lees_edwards.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+from __future__ import print_function
+
+import espressomd as md
+import unittest as ut
+
+@ut.skipIf(not md.has_features(['LEES_EDWARDS']),
+'Features not available, skipping test!')
+
+class Lees_Edwards_test(ut.TestCase):
```
Comment (KaiSzuttor) [inline testsuite/lees_edwards.py]: please use pep8 naming convention for classes: CamelCase

File: testsuite/lees_edwards.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+  def test(self):
+
```
Comment (KaiSzuttor) [inline testsuite/lees_edwards.py]: Please provide a docstring for the test method that describes what is beeing tested and how.

File: testsuite/lees_edwards.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      
+      self.assertLess(pos_difference, tol)
+      self.assertLess(vel_difference, tol)
+    
+    for i in range(3):
+      system.integrator.run(1)
+      system.lees_edwards_offset = 0.1
+      pos_difference = system.part[0].pos_folded[0] - 2.6
+      vel_difference = system.part[0].v[1] - -0.1
+      print(pos_difference, vel_difference)
```
Comment (KaiSzuttor) [inline testsuite/lees_edwards.py]: please remove the prints

File: testsuite/lees_edwards.py (hunk 2 above)
Comment (KaiSzuttor) [reply testsuite/lees_edwards.py]: Maybe devide into several tests for clarification.

File: testsuite/lees_edwards.py [hunk 4]
```diff
@@ -1,34 +1,42 @@
...
 import espressomd as md
 import unittest as ut
 
+
 @ut.skipIf(not md.has_features(['LEES_EDWARDS']),
-'Features not available, skipping test!')
+  'Feature not available, skipping test!')
 
-class Lees_Edwards_test(ut.TestCase):
+class lees_edwards_test(ut.TestCase):
```
Comment (KaiSzuttor) [inline testsuite/lees_edwards.py]: Sorry that is still wrong... it should be named "LeesEdwardsTest" or similar.

Comment (bindgens1) [reply testsuite/lees_edwards.py]: Sorry for the effort, should be correct now.

**Pull request merged**